pkg/projectfile/projectfile.go

```diff
@@ -391,14 +390,14 @@ func MakeEventsFromConstrainedEntities(items []ConstrainedEntity) (events []*Eve
 
 // Script covers the script structure, which goes under Project
 type Script struct {
-	Name        string            `yaml:"name"`
-	Description string            `yaml:"description,omitempty"`
-	Value       string            `yaml:"value"`
-	Filename    string            `yaml:"filename,omitempty"`
-	Standalone  bool              `yaml:"standalone,omitempty"`
-	Language    language.Language `yaml:"language,omitempty"`
-	Conditional Conditional       `yaml:"if"`
-	Constraints Constraint        `yaml:"constraints,omitempty"`
+	Name        string      `yaml:"name"`
+	Description string      `yaml:"description,omitempty"`
+	Value       string      `yaml:"value"`
+	Filename    string      `yaml:"filename,omitempty"`
+	Standalone  bool        `yaml:"standalone,omitempty"`
+	Language    string      `yaml:"language,omitempty"`
```

Comment:
Why not `[]language.Language` ?

Reply:
This is to avoid breaking changes to the `activestate.yaml`. If we use `[]language.Language` then the field value has to become a list and we have to represent it with `[entry, entry]` or 

```
list:
  - entry
  - entry
```